.github/workflows/generate-changelog.yml

```diff
@@ -13,11 +13,48 @@ jobs:
     with:
       script_call: 'make tools update-changelog-unreleased-section'
       file_to_commit: 'CHANGELOG.md'
-      commit_message: ${{ github.event_name == 'workflow_dispatch' && 'Update CHANGELOG.md (Manual Trigger)' || format('{0}{1}', 'Update CHANGELOG.md for \#', github.event.pull_request.number) }}
+      commit_message: ${{ github.event_name == 'workflow_dispatch' && format('{0}{1}', 'chore:',' Updates CHANGELOG.md (Manual Trigger)') || format('{0}{1}{2}{3}', 'chore:', ' Updates CHANGELOG.md for ', '#', github.event.pull_request.number) }}
```

Comment:
can we adjust the commit message in the release process to include the chore prefix as well? All good if you prefer to handle in separate PR.

Reply:
I will include it in this PR. Thanks for the reminder

Reply:
Done